staff/daniel-toledo/gamma-template/style.sass

```diff
@@ -0,0 +1,419 @@
+*
+    margin: 0
+    padding: 0
+    box-sizing: border-box
+
+%green
```

Comment:
cool use of sass features! Also, to keep only the color you could have used a variable, $green: rgb...

Reply:
Nice Idea!! changed it